### PR TITLE
(docs) - Badge tweaks

### PR DESCRIPTION
**Pull request by andyrichardson**
## About

 - Replace offensive spectrum badge 😅 
 - Use [badgen](https://badgen.net/) for all badges (consistent colors, better perf than shields.io)
 - Flaunt that MIT
 - Show total downloads
 - Add link to docs

[Readme preview](https://github.com/FormidableLabs/urql/blob/less-offensive-badges/README.md)

### CODE REVIEW

**Comment by wgolledge** [comment]
I think this looks awesome. That branch name 😂

**Comment by andyrichardson** [comment]
> I think this looks awesome. That branch name 😂

It's just that spectrum badge - it hits you like a slap in the face 👋

File: README.md [hunk 1]
```diff
@@ -11,14 +11,20 @@
   <br />
   <br />
   <a href="https://circleci.com/gh/FormidableLabs/urql">
-    <img alt="Test Status" src="https://circleci.com/gh/FormidableLabs/urql.svg?style=shield" />
-  </a> <a href="https://github.com/FormidableLabs/urql#maintenance-status">
-    <img alt="Maintenance Status" src="https://img.shields.io/badge/maintenance-active-green.svg" />
```
Comment (kitten) [inline README.md:16]: This badge should be retained on every Formidable repo :)

File: README.md [hunk 2]
```diff
@@ -11,14 +11,20 @@
...
+    <img alt="Total downloads" src="https://badgen.net/npm/dt/urql?color=blue" />
+  </a>
+  <a href="https://formidable.com/open-source/urql/docs/">
+    <img alt="Spectrum badge" src="https://badgen.net/badge/docs/visit site/orange" />
   </a>
   <a href="https://spectrum.chat/urql">
-    <img alt="Spectrum badge" src="https://withspectrum.github.io/badge/badge.svg" />
+    <img alt="Spectrum badge" src="https://badgen.net/badge/spectrum/chat with us/purple" />
+  </a>
+  <a href="https://github.com/FormidableLabs/urql/blob/master/LICENSE">
```
Comment (kitten) [inline README.md:25]: GitHub now displays this in the repo info header, so I think we won’t have to duplicate this 👀

File: README.md [hunk 3]
```diff
@@ -11,14 +11,20 @@
...
-    <img alt="Test Status" src="https://circleci.com/gh/FormidableLabs/urql.svg?style=shield" />
-  </a> <a href="https://github.com/FormidableLabs/urql#maintenance-status">
-    <img alt="Maintenance Status" src="https://img.shields.io/badge/maintenance-active-green.svg" />
+    <img alt="Test Status" src="https://badgen.net/github/status/FormidableLabs/urql/master/ci/circleci?label=circleci" />
+  </a>
+  <a href="https://www.npmjs.com/package/urql">
+    <img alt="Total downloads" src="https://badgen.net/npm/dt/urql?color=blue" />
+  </a>
+  <a href="https://formidable.com/open-source/urql/docs/">
+    <img alt="Spectrum badge" src="https://badgen.net/badge/docs/visit site/orange" />
```
Comment (kitten) [inline README.md:20]: The alt tag here is off :)

File: README.md [hunk 4]
```diff
@@ -11,14 +11,20 @@
   <br />
   <br />
   <a href="https://circleci.com/gh/FormidableLabs/urql">
-    <img alt="Test Status" src="https://circleci.com/gh/FormidableLabs/urql.svg?style=shield" />
-  </a> <a href="https://github.com/FormidableLabs/urql#maintenance-status">
-    <img alt="Maintenance Status" src="https://img.shields.io/badge/maintenance-active-green.svg" />
+    <img alt="Test Status" src="https://badgen.net/github/status/FormidableLabs/urql/master/ci/circleci?label=circleci" />
+  </a>
+  <a href="https://www.npmjs.com/package/urql">
+    <img alt="Total downloads" src="https://badgen.net/npm/dt/urql?color=blue" />
```
Comment (JoviDeCroock) [inline README.md:17]: I'm not entirely sure how valuable total downloads is as compared to monthly. Personally not a huge fan of adding the downloads in there

Comment (kitten) [reply README.md:17]: Yea, it comes across as gloating at times. Which is fine at times I believe, we can celebrate our successes 😅 but I agree that monthly/weekly downloads are better

File: README.md (hunk 3 above)
Comment (andyrichardson) [reply README.md:20]:
```suggestion
    <img alt="Visit docs" src="https://badgen.net/badge/docs/visit site/orange" />
```

File: README.md (hunk 2 above)
Comment (andyrichardson) [reply README.md:25]: My thinking was "if you've got it, flaunt it"! I'll remove it anyhow

File: README.md (hunk 1 above)
Comment (andyrichardson) [reply README.md:16]: Is that a thing we do? 

I don't see what value it provides for actively maintained projects - a [deprecation notice](https://github.com/andyrichardson/atom-updater-linux) when a project is no longer maintained however 👌

Comment (kitten) [reply README.md:16]: Yep, it's been added to all our packages; here's one of the PRs: https://github.com/FormidableLabs/urql/pull/226
Those correspond to our "OSS Maintenance Levels" to transparently communicate which projects are also stable or experimental. Deprecated (or "Archived" in our case) is just one of four statuses.

File: README.md (hunk 4 above)
Comment (andyrichardson) [reply README.md:17]: I think total downloads is a nice metric for tracking milestones

Comment (andyrichardson) [reply README.md:17]:
```suggestion
    <img alt="Total downloads" src="https://badgen.net/npm/dm/urql?color=blue" />
```

**Pull request merged**